src/panels/README.md

```diff
@@ -1,27 +1,48 @@
 # Visual Studio Code Panels
 
-The `vscode-panels` component is a web component implementation of a [tab](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role). The component is created using three components that work together to interchangably display different content:
+The `vscode-panels` component is a web component implementation of a [tab](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role).
+
+![Panel hero](/docs/assets/images/panels-hero.png)
+
+The component is created using three components that work together to interchangably display different content:
 
 -   `<vscode-panels>`: The top level container element.
 -   `<vscode-panel-tab>`: Renders the panel tab that will be associated with a panel view.
 -   `<vscode-panel-view>`: The container element that will hold content associated with a given tab.
 
-## Panels Attributes
+## Usage
```

Comment:
An example for overflow when the screen becomes narrow? I assume we just roll things into a `...`, thought not sure if that works out of the box.

Reply:
The panels component will never overflow in small screens/doesn't really adapt to them in general 😅. Panels will never get smaller than the min-width of it's longest content (which is typically the length of all the tabs).

This exists because we got dinged in our a11y review for not being allowed to have text use `...`––instead, text must always be fully visible.

In practical terms, however, I'm hoping (🤞🤞🤞) this doesn't make too big a difference because I don't know many/anyone who's using VS Code at this width:

<img width="1287" alt="Screen Shot 2022-02-04 at 11 59 22 AM" src="https://user-images.githubusercontent.com/39639992/152595175-479a6f05-55fd-4144-a014-45fa1d9aafbf.png">